### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by GMartinez-Sisti**

### CODE REVIEW

File: fyde-access-proxy/ha.md [hunk 1]
```diff
@@ -18,6 +18,6 @@ nav_order: 9
 
 - Please check the [Network]({{ site.baseurl }}{% link fyde-access-proxy/req-net.md %}) requirements for load balancer information and diagram
 
-- NOTE: Redis Replication is out of the scope for this document
+- Redis Replication is out of the scope for this document
```
Comment (rasmartins) [inline fyde-access-proxy/ha.md]: What about *of* this document ?

File: fyde-access-proxy/req-net.md [hunk 2]
```diff
@@ -40,7 +40,8 @@ NOTE: assuming default values
 
 ### Highly Available mode
 
-- NOTE: Redis Replication is out of the scope for this document
-  - Please check `https://redis.io/topics/replication`
+- Redis Replication is out of the scope for this document
```
Comment (rasmartins) [inline fyde-access-proxy/req-net.md]: Same thing with the of instead of for.

File: quickstart.md [hunk 3]
```diff
@@ -7,62 +7,28 @@ nav_order: 2
...
 
 - Add the following:
 
-- **Proxy Name**: Used to identify the proxy
-- **Location**: Optional, for your reference only
-- **Host**: IP/DNS record used to reach the proxy
-- **Port**: Port where the Fyde Access Proxy will be available
+  - **Proxy Name**: Used to identify the proxy
+  - **Location**: Optional, for your reference only
+  - **Host**: IP/DNS record used to reach the proxy
```
Comment (rasmartins) [inline quickstart.md]: I would use "host name" instead of IP/DNS

File: fyde-access-proxy/ha.md (hunk 1 above)
Comment (GMartinez-Sisti) [reply fyde-access-proxy/ha.md]: You're right. This has been copied around without proper proofing. Changing.

File: fyde-access-proxy/req-net.md (hunk 2 above)
Comment (GMartinez-Sisti) [reply fyde-access-proxy/req-net.md]: Changed.

File: quickstart.md (hunk 3 above)
Comment (GMartinez-Sisti) [reply quickstart.md]: Makes sense. Changed.

**Pull request merged**